index.html

```diff
@@ -33,7 +99,36 @@
           r.revision.push(entry.revision.substr(0, 6));
         }
       }
-      return res
+      return res;
+    }
+
+    function copy_to_clipboard(clipboardInputElem, text) {
+      clipboardInputElem.textContent = text;
+      clipboardInputElem.select();
+      clipboardInputElem.setSelectionRange(0, 99999); /* For mobile devices */
+
+      document.execCommand("copy");
+    }
+
+    function show_clipboard_text(commit_hash) {
+      var notifyClipboardTextElem = document.getElementById("notify-clipboard-text");
+      notifyClipboardTextElem.textContent = commit_hash + " copied to clipboard";
+      notifyClipboardTextElem.className = 'inline-text';
+      // notifyClipboardTextElem.className = 'hidden';
+      // notifyClipboardTextElem.textContent = commit_hash + " copied to clipboard";
+      // notifyClipboardTextElem.className = 'clipboard-text-animated';
+      setTimeout(function () {
+        notifyClipboardTextElem.className = 'hidden';
+      }, 3000);
+    }
+
+    function data_point_click(data) {
+      var commit_hash = data.points[0].x;
+      var clipboardInputElem = document.getElementById("clipboard-input");
```

Comment:
```js
        plotDiv.on('plotly_click', data => {
          navigator.clipboard.writeText(data.points[0].x);
        });
```

works on my machine™. We could also request the `clipboard-write` permission, but:

 - it's not supported in Firefox
 - it's not needed in Chrome

So I guess there's no need for that.

Reply:
Changed to use clipboard.writeText function as it works on Chrome, Firefox and Edge